index.html

```diff
@@ -13,19 +13,34 @@
     <div id="target-two">
       <div class="bg-blue-500 p-32"><h1>Content Two</h1></div>
     </div>
-    <button id="button-two">Toggle two</button>
+    <div class="flex flex-col">
+      <button id="button-two">Toggle two</button>
+      <button id="expand-one">Expand one</button>
+      <button id="collapse-one">Collapse one</button>
+    </div>
     <script type="module">
       import * as operations from './index.js'
 
       let twoIsCollapsed = false
+      const one = document.querySelector('#target-one')
       const two = document.querySelector('#target-two')
 
       function toggleTwo() {
         operations[twoIsCollapsed ? 'expand' : 'collapse'](two)
         twoIsCollapsed = !twoIsCollapsed;
       }
 
+      async function expandOne() {
+        await operations.expand(one)
+        console.log("Expand completed")
+      }
+      function collapseOne() {
+        operations.collapse(one)
```

Comment:
Do we also need to await collapse here?

Reply:
No need, not production code!

Reply:
I don't understand what awaiting collapse have to do with production code 😅

Reply:
Awaiting here would be a stylistic choice since we don't have anything afterward. If we feel users might see this and be confused we can totally add it, but it wouldn't really _do_ anything.